lapce-ui/src/app.rs

```diff
@@ -83,6 +83,15 @@ where
         .title(LocalizedString::new("Lapce").with_placeholder("Lapce"))
         .window_size(size)
         .set_position(pos);
+
+    #[cfg(target_os = "windows")]
+    if Config::load(&lapce_data::data::LapceWorkspace::default())
```

Comment:
You can pass the Config in from the parameter.

Reply:
done